src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBOptions.java

```diff
@@ -25,18 +25,24 @@ public GenomicsDBOptions(final Path reference) {
     }
 
     public GenomicsDBOptions(final Path reference, GenomicsDBArgumentCollection genomicsdbArgs) {
-        this(reference, genomicsdbArgs, new GenotypeCalculationArgumentCollection(), false);
+        this(reference, genomicsdbArgs, new GenotypeCalculationArgumentCollection());
     }
 
     public GenomicsDBOptions(final Path reference, final GenomicsDBArgumentCollection genomicsdbArgs,
```

Comment:
Bleh, GenomicsDBOptions and GenomicsDBArgumentCollection are confusingly similar.  We should probably revisit that at some point.

Reply:
Yes, we should. As of now, [GenomicsDBOptions](https://github.com/broadinstitute/gatk/blob/3924f9a9833afd57ece41f6c0d9207794fd19268/src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBOptions.java#L31) is a combination of `GenomicsDBArgumentCollection` and `GenotypeCalculationArgumentCollection`.